beanstalk_worker/Dockerfile.workers

```diff
@@ -1,14 +1,17 @@
 FROM registry.centos.org/centos/centos
 MAINTAINER Bamacharan Kundu<bkundu@redhat.com>
+
 RUN yum update -y && \
     yum install python -y && \
+    yum install PyYAML -y && \
```

Comment:
@navidshaikh :+1:  for adding ``PyYAML``. But in doing that you broke https://github.com/CentOS/container-pipeline-service/blob/master/beanstalk_worker/worker_start_build.py#L163, which was written before adding PyYAML as a dependency. I do not see how it was running in your devcloud, because it did not run in mine. I am fixing it for now.